packages/dashboard/src/components/DashboardGridLayout/DashboardGridLayoutWithAddCardPanel/DashboardGridLayoutWithAddCardPanel.tsx

```diff
@@ -47,6 +47,40 @@ export class DashboardGridLayoutWithAddCardPanel extends BaseComponent<
     };
   }
 
+  public componentDidMount(): void {
+    if (this._cardsForAddCardPanel !== this.props.addCardPanelCards || this._cardsForLayout !== this.props.dashboardCards) {
+      this._cardsForAddCardPanel = this.props.addCardPanelCards;
+      this._cardsForLayout = this.props.dashboardCards;
+      const cardIds: string[] = [];
+      const cardNodes: JSX.Element[] = [];
+      const layout: DashboardGridBreakpointLayouts = { lg: [{ i: 'section0', y: 0, x: 0, size: CardSize.section }] };
+      layout.lg = layout.lg!.concat(this.props.layout.lg!);
+      this.props.dashboardCards.map((card: IDGLCard) => {
```

Comment:
Its preferred to use `.forEach` rather than `.map` if you aren't doing anything with the array that `.map` allocates.

See:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

vs.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach